sites/partners/pages/listings/[id]/applications/index.tsx

```diff
@@ -27,6 +27,11 @@ import { AgGridReact } from "ag-grid-react"
 import { getColDefs } from "../../../../src/applications/ApplicationsColDefs"
 import { GridOptions, ColumnApi, ColumnState } from "ag-grid-community"
 
+type ApplicationsListSortOptions = {
+  orderBy: string
```

Comment:
Can be typed with EnumApplicationsApiExtraModelOrderBy and EnumApplicationsApiExtraModelOrder now

Reply:
I an use `EnumApplicationsApiExtraModelOrder ` only, because `EnumApplicationsApiExtraModelOrderBy ` has different values as we use in the sorting controller